openstack/standalone-cinder/pkg/provisioner/provisioner.go

```diff
@@ -142,6 +164,9 @@ ERROR_UNRESERVE:
 	}
 	glog.V(3).Infof("Volume %s unreserved", volumeID)
 ERROR_DELETE:
+	if found {
+		goto ERROR
+	}
```

Comment:
I would prefer this to read:

if !found {
  // Delete volume
}

Instead of adding a goto.  The gotos are readable only if there is a single stack of operations that are unwound by the error cleanup labels at the end.  As soon as you add additional jumps it gets harder to follow.